config/defaults.hjson

```diff
@@ -75,6 +75,8 @@
       "ProxyAllImages"
     # Timeout for uploading images to pictrs (in seconds)
     upload_timeout: 30
+    # Resize post thumbnails to this maximum width/height.
+    max_thumbnail_size: 100
```

Comment:
The default thumbnail size in lemmy-ui is `256` , might be good to use that.

Reply:
Changed